src/store/promoter/__mocks__/promoterState.ts

```diff
@@ -0,0 +1,21 @@
+export default {
+    selection: {
+        boundingRect: {
+            height: 10,
+            width: 10,
+            type: 'rect' as const,
+            x: 100,
```

Comment:
This somewhat implies that these values are percentages like other `Rect` objects, but that's not the case, correct? Can we make these mock values more pixel-y?